lm_eval/base.py

```diff
@@ -152,7 +152,7 @@ def tok_decode(self, tokens: Iterable[int]):
         pass
 
     @abstractmethod
-    def _model_generate(self, context, max_length, eos_token_id):
+    def _model_generate(self, context, max_length, eos_token_id, top_k=2, do_sample=False):
```

Comment:
Are these changes MLSUM specific? I am worried that this might break other tasks.

Reply:
These are required for open generation tasks that call generation function of the LM. They are default arguments, so they should not break other tasks.